src/pages/components/SearchList/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from "react";
+import { ScrollView, TouchableOpacity, View, Text, StyleSheet } from "react-native";
+import SearchInput, { createFilter } from "react-native-search-filter";
+
+export default function SearchList({data, searchFields, visibleKeys, titleStyle, dataStyle}) {
+  const [searchItem, setSearchItem] = useState("");
+
+  const filterData = data.filter(
+    createFilter(searchItem, searchFields)
+  )
+
+  const keys = visibleKeys || Object.keys(data[0] || []);
+
+  return (
+    <View>
+        <View style={{ flexDirection: "row", justifyContent: "space-around" }}>
```

Comment:
try to extract all varying details like search placeholder, style of containers/texts  to be coming from props

Reply:
added inputPlaceholder, flexWidth, numberOfLines, other styling stuffs to be coming from props